docs/video-player.md

```diff
@@ -0,0 +1,128 @@
+
+The GraphQL query returns data that the `VideoPlayer` component automatically uses to properly size the player, set up a “blur-up” placeholder as well as lazy loading the video.
+
+## Example
+
+For a fully working example take a look at our [examples directory](https://github.com/datocms/react-datocms/tree/master/examples).
+
+```js
+import React from 'react';
+import { VideoPlayer } from 'react-datocms';
+
+const Page = ({ data }) => (
+  <div>
+    <h1>{data.blogPost.title}</h1>
+    <VideoPlayer data={data.blogPost.cover.video} />
+  </div>
+);
+
+const query = gql`
+  query {
+    blogPost {
+      title
+      cover {
+        video {
+          # required: this field identifies the video to be played
+          muxPlaybackId
+
+          # all the other fields are not required but:
+
+          # if provided, title is displayed in the upper left corner of the video
+          title
+
+          # if provided, width and height are used to define the aspect ratio of the
+          # player, so to avoid layout jumps during the rendering.
+          width
+          height
+
+          # if provided, it shows a blurred placeholder for the video
+          blurUpThumb
+
+          # you can include more data here: they will be ignored by the component
+        }
+      }
+    }
+  }
+`;
+
+export default withQuery(query)(Page);
+```
+
+## Props
+
+The `<VideoPlayer />` components supports all the properties made available for
+`<MuxPlayer />` component from `@mux/mux-player-react` package plus `data`, which is meant to
+receive data directly in the shape they are provided by DatoCMS GraphQL API.
+
+`<Video Player />` uses the `data` prop to generate a set of props for the inner
+`<MuxPlayer />`. On this topic, also see the "Advanced usage" section below.
+
+| prop | type           | required           | description                                                      | default |
+| ---- | -------------- | ------------------ | ---------------------------------------------------------------- | ------- |
+| data | `Video` object | :white_check_mark: | The actual response you get from a DatoCMS `video` GraphQL query |         |
+
+Compared to the `<MuxPlayer />`, some prop default values are different on `<VideoPlayer />`
+
+- `disableCookies` is normally true, unless you explicitly set the prop to `false`
+- the video height and width, when available in the `data` props, are used to set `{ aspectRatio: "[width] / [height]"}` for the `<MuxPlayer />`'s `style`
+
+All the other props are forwarded to the `<MuxPlayer />` component that is used internally.
+
+## Advanced usage: the `useVideoPlayer` hook
+
+Despite we try our best to make the `<VideoPlayer />` suitable and easy to use for most normal use cases, there are situation where you may need to leverage the `<MuxPlayer />` directly. If that's the case, fill free to use the hook we provide: `useVideoPlayer`.
```

Comment:
"situations" (plural) instead of situation.

Also, do we have any easy examples? Just 1-2 sentences about why someone might want to fall back to the MuxPlayer instead of ours?

(It's not super clear to me what ours even does... does it mostly just[ destructure our GraphQL query into MuxPlayer-compatible props](https://github.com/datocms/react-datocms/pull/90/files#diff-d8f40cc4513f251bf681c00a75927bb315ab3a624466fe36da9b7b4c6be54d53R78-R85)?

Reply:
Yes, @arcataroger, the hook is quite simple, but it can be handy anyway because it knows how to treat data directly in the shape they come from GraphQL API: so, one less thing to think about for developers.

Reply:
I totally agree that it's a useful function, just wasn't sure if it should be a hook or just a regular function.

But sorry, this isn't something I'm an expert on, and I'm not confident enough about it to argue it one way or the other :) Your judgment call! Or maybe see what the other devs think?

It's not a big deal; either way, it's a useful thing to offer, whatever we call it!